srcpkgs/proton-bridge/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'proton-bridge'
+pkgname=proton-bridge
+version=1.2.7
+revision=1
+archs="*"
+wrksrc="proton-bridge-${version}-live.1"
+#create_wrksrc=yes
+build_style=gnu-makefile
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends=""
+makedepends="binutils git go>=1.13 libsecret-devel pkg-config qt5-go tar which"
```

Comment:
`binutils` `go` `pkg-config` `qt5-go` `tar` `which` should all go in `hostmakedepends`